packages/playwright/src/browser.ts

```diff
@@ -36,8 +37,12 @@ export const axeRunPartial = ({
 };
 
 export const axeFinishRun = ({
-  partialResults,
   options
 }: FinishRunParams): Promise<AxeResults> => {
-  return window.axe.finishRun(partialResults, options);
+  return window.axe.finishRun(JSON.parse(window.partialResults), options);
```

Comment:
Why can't `partialResults` be passed in here?

Reply:
That was the problem. It was too big to be put here on it's own